rpcs3/rpcs3qt/user_create_dialog.cpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "user_create_dialog.h"
+
+user_create_dialog::user_create_dialog(const std::string& dir, QWidget* parent)
+	: QDialog(parent)
+{
+	setWindowTitle(tr("New User"));
+	setMinimumSize(QSize(50, 100));
+
+	// Create buttons
+	m_okButton = new QPushButton(tr("Ok"), this);
+	m_cancelButton = new QPushButton(tr("Cancel"), this);
+	m_textBox = new QLineEdit();
+
+	// Set constraints on textbox
+	m_textBox->setMaxLength(16);
+
+	// Button Layout
+	QHBoxLayout* hbox_buttons = new QHBoxLayout();
+	hbox_buttons->addWidget(m_okButton);
+	hbox_buttons->addWidget(m_cancelButton);
+
+	// main layout
+	QVBoxLayout* vbox_main = new QVBoxLayout();
+	vbox_main->setAlignment(Qt::AlignCenter);
+	vbox_main->addWidget(m_textBox);
+	vbox_main->addLayout(hbox_buttons);
+	setLayout(vbox_main);
+
+	connect(m_cancelButton, &QAbstractButton::clicked, this, &user_create_dialog::close);
```

Comment:
add
`connect(m_okButton, &QAbstractButton::clicked, this, &user_create_dialog::<your_signal>);`
(see other comment)